### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by Leboroz**
## Features added: ♻️ 
- I added a Webpack configuration.
- Sections added:
  - Header
  - content
-  I styled my sections with sass.

### CODE REVIEW

File: src/modules/components/header.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function header() {
+  const header = document.createElement('section');
+  header.className = 'title padding-x';
+  header.innerHTML = `
+  <div class="header">
+    <h1 class="display-1">Leaderboard</h1>
+  </div>`;
+
+  return header;
+}
```
Comment (Lameck1) [inline src/modules/components/header.js]: - [x] Let's remind ourselves that we are working with `webpack` and `ES6` features in this project. How about refactoring this function to using the ES6 syntax.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Leboroz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/components/header.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const header = () => {
```
Comment (mateo951) [inline src/modules/components/header.js:1]: Addressed 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @Leboroz, 

Great job making the changes requested by your previous reviewer 😁
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**